internal/flag/flag.go

```diff
@@ -327,9 +332,15 @@ func Yes() Bool {
 		Name:        flagnames.Yes,
 		Shorthand:   "y",
 		Description: "Accept all confirmations",
+		Aliases:     []string{"auto-confirm"},
 	}
 }
 
+// Checks whether the --yes flag or the --auto-confirm flag is present
+func HasYes(ctx context.Context) bool {
```

Comment:
The convention we have been following so far is to call the function `flag.GetFlagName()` for `flag.FlagName()`, in this case it should be `flag.GetYes()`